hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/client/OzoneBucketStub.java

```diff
@@ -273,14 +371,14 @@ public OmMultipartUploadCompleteInfo completeMultipartUpload(String key,
       ByteArrayOutputStream output = new ByteArrayOutputStream();
 
       int prevPartNumber = 0;
-      for (Map.Entry<Integer, String> part: partsMap.entrySet()) {
+      for (Map.Entry<Integer, String> part : partsMap.entrySet()) {
```

Comment:
Please don't have whitespace change like this.  Since this is an existing file in the master branch, it will make the merge harder.